File: front-end/src/app/models/solicitacao.ts
import {Prestador} from "./prestador";
import {Usuario} from "./usuario";

export class Solicitacao {

  id: number ;
  prestadorID: number ;
  prestador:Prestador;
  usuario:Usuario;
  usuarioID: number ;
  data_utilizacao: Date ;
  status: string ;
  valor_final:number






  public constructor(data:any) {
    if (data) {
      this.id = Number(data.id);


    }else{
      this.id = 0;


    }
  }
}

File: back-end/help-car/model/Solicitacao.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace help_car.model
{
    [Table("tbsolicitacao")]
    public class Solicitacao
    {
        [Column("id")]
        public int? id { get; set; }
        public DateTime? data_utilizacao { get; set; }
        public Double? valor_final { get; set; }
        

        public int? usuarioID { get; set; }   
        public int? prestadorID { get; set; }
  
        public string? status { get; set; }
    }
}
